### DISCUSSION TITLE
Is there any specific usage tutorial for pyslurm 20.11.8 version?

### DISCUSSION THREAD

**Discussion by zhaoshuyi12** [PySlurm/pyslurm#358] [Q&A]
I think it is currently higher than this version.

**Comment by tazend** [comment]
Hi,

unfortunately, there is not really an elaborate tutorial for this Version. Starting for Slurm major versions 21.8, a new API was introduced, deprecating the old one, and also improving everything including documentation from there on. It will likely not be backported to versions before 21.8, because the API changes are too big in these old Slurm releases, and the time invest would be too high at the moment.
